### DISCUSSION TITLE
DenseAmpcor used in Dense offset calculation?

### DISCUSSION THREAD

**Discussion by riyasmjgeo** [isce-framework/isce2#519] [Q&A] [answered]
I would like to clarify that while calculating the _dense offset_, is the function DenseAmpcor.py used for it? 
If so, is there any way to address the application to perform autoRIFT instead of DenseAmpcor (in the input XML probably)?

**Comment by EJFielding** [comment, marked as answer]
Yes, the `dense offset` options in `topsApp.py` and `stripmapApp.py` both use the ISCE2 `Ampcor` calculation. Those applications do not have an option to run autoRIFT.

**Reply by riyasmjgeo** [reply]
Thank you for the response. I got confused with another program `denseOffsets.py` in ISCE.
Is there any document/ tutorial available to refer the usage of autoRIFT in ISCE2 package?

**Reply by SarsijNayan** [reply]
Hey, please help me to use autorift.
